core/src/main/scala/org/apache/spark/metrics/ExecutorMetricType.scala

```diff
@@ -59,6 +59,18 @@ case object JVMOffHeapMemory extends ExecutorMetricType {
   }
 }
 
+case object ProcessTreeRSSMemory extends ExecutorMetricType {
```

Comment:
could you separate out memory from python, the jvm, and other here as well?  I think that breakdown is really important for most users.  (can also be a follow-up)

Reply:
I will try to see what can I do.